sopt/src/main/scala/dagr/sopt/ArgTokenizer.scala

```diff
@@ -41,13 +44,15 @@ object ArgTokenizer {
 
 /** A class to tokenize a sequence of strings prior to option parsing.  A series of tokens will
   * be available.  If any errors are encountered, a failure will be present instead of the token.  */
-class ArgTokenizer(args: TraversableOnce[String]) extends Iterator[Try[ArgTokenizer.Token]] {
+class ArgTokenizer(val argFilePrefix: Option[String] = None,
+                   args: Seq[String]) extends Iterator[Try[ArgTokenizer.Token]] {
   import ArgTokenizer._
 
   private var nextToken: Option[Try[Token]] = None
-  private val iterator = args.toIterator
+  private val stack = scala.collection.mutable.Stack[String](args:_*)
```

Comment:
I prefer putting `import scala.collection.mutable` in the list of imports and then putting `mutable.Stack...`.  It makes it very clear you are choosing `mutable` and `Stack` without having to read through `scala.collection`.